packages/desktop-client/src/components/accounts/Header.jsx

```diff
@@ -130,6 +130,33 @@ export function AccountHeader({
     },
     [searchInput],
   );
+  useHotkeys(
+    'a',
+    () => onAddTransaction(),
+    {
+      preventDefault: true,
+      scopes: ['app'],
+    },
+    [onAddTransaction],
+  );
+  useHotkeys(
+    'i',
+    () => onImport(),
+    {
+      scopes: ['app'],
+    },
+    [onImport],
+  );
+  useHotkeys(
+    'ctrl+b, cmd+b, meta+b',
```

Comment:
I think we can clean this up a bit based on [the documentation,](https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/basic-usage#modifiers--special-keys)
```suggestion
    'mod+b',
```

Reply:
No, that is not the same set of keys.  `mod` is just `ctrl` and `cmd`.  You lost `meta`.  So at best we could go to `mod+b, meta+b` but that isn't much shorter and, IMO, is much harder to quickly read.  You have to *know* what `mod` means in that library.  Explicitly listing them, you do not need to know that.